code/modules/integrated_electronics/subtypes/converters.dm

```diff
@@ -161,6 +161,7 @@
 		var/I = get_pin_data(IC_INPUT, k)
 		if(!isnull(I))
 			result = result + I
+			result = copytext(1, MAX_MESSAGE_LEN)
```

Comment:
The lag comes as a result of large messages being combined. Putting a limit after the text is already combined doesn't stop the exploit from happening.

Reply:
yes it does